src/main/pages/che-7/administration-guide/proc_assigning-che-permissions.adoc

```diff
@@ -22,10 +22,10 @@ The following is a message body that requests permissions for a user with a `use
     "configure",
     "setPermissions"
   ],
-  "userID": "userID",          <1>
+  "userId": "userId",          <1>
```

Comment:
Only the parameter name must remain unchanged. The value stand-in should be consistent with the rest of the docs.
```suggestion
  "userId": "userID",          <1>
```